### PR TITLE
remove unneccessary address in block

### PR DESCRIPTION
**Pull request by n-hutton**

### CODE REVIEW

File: libs/ledger/src/execution_manager.cpp [hunk 1]
```diff
@@ -177,7 +177,7 @@ ExecutionManager::ScheduleStatus ExecutionManager::Execute(Block const &block)
   // update the last block hash
   state_.ApplyVoid([&block](Summary &summary) {
     summary.last_block_hash   = block.hash;
-    summary.last_block_miner  = block.miner;
+    summary.last_block_miner  = chain::Address(block.miner_id);
```
Comment (n-hutton) [inline libs/ledger/src/execution_manager.cpp:180]: Is this neccessary?

**Review by ejfitzgerald — COMMENTED** [review]
Can you double check that fees are not affected by this change, but otherwise looks good

File: libs/network/include/network/p2pservice/p2p_http_interface.hpp [hunk 2]
```diff
@@ -155,7 +155,7 @@ class P2PHttpInterface : public http::HTTPModule
       block["hash"]         = "0x" + b->hash.ToHex();
       block["previousHash"] = "0x" + b->previous_hash.ToHex();
       block["merkleHash"]   = "0x" + b->merkle_hash.ToHex();
-      block["miner"]        = b->miner.display();
+      block["miner"]        = b->miner_id.identifier().ToBase64();
```
Comment (ejfitzgerald) [inline libs/network/include/network/p2pservice/p2p_http_interface.hpp]: This needs to be the address (block explorer needs it to be)

**Review by ejfitzgerald — CHANGES_REQUESTED** [review]

File: libs/network/include/network/p2pservice/p2p_http_interface.hpp (hunk 2 above)
Comment (n-hutton) [reply libs/network/include/network/p2pservice/p2p_http_interface.hpp]: Ok

**Review by ejfitzgerald — APPROVED** [review]

**Pull request merged**